libs/service-portal/health/src/screens/Vaccinations/VaccinationsOther.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, SkeletonLoader } from '@island.is/island-ui/core'
+import { HealthPaths } from '../../lib/paths'
+import { VaccinationsWrapper } from './wrapper/VaccinationsWrapper'
+import { otherVaccinationsData } from './dataStructure'
+import { SortedVaccinationsTable } from './tables/SortedVaccinationsTable'
+
+export const VaccinationsOther = () => {
+  const loading = false
+  const data = otherVaccinationsData
+  // Fetch data here
+
+  return (
+    <VaccinationsWrapper pathname={HealthPaths.HealthVaccinationsOther}>
+      <Box>
+        {loading && (
+          <SkeletonLoader
+            repeat={3}
+            space={2}
+            height={24}
+            borderRadius="standard"
+          />
+        )}
+        <SortedVaccinationsTable data={data} />
+      </Box>
+    </VaccinationsWrapper>
+  )
+}
+
```

Comment:
This seems like an exact copy of the other screen? Why?

Reply:
Rendering different paths and fetching different data. The wrapper has most code but the loader is dependant on the fetching.